src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -236,8 +236,9 @@ class TypeSpecTest {
         .addFun(FunSpec.builder("fooBar")
             .addModifiers(KModifier.PUBLIC, KModifier.ABSTRACT)
             .addAnnotation(AnnotationSpec.builder(headers)
-                .addMember("value", "%S", "Accept: application/json")
-                .addMember("value", "%S", "User-Agent: foobar")
+                .addMemberArrayElement("value", Array<String>::class, "%S",
```

Comment:
Since we never have to specify the generic type, we only distinguish between primitive type arrays and all other arrays. I was thinking about better API to avoid passing unnecessary params, is this a good idea?